default/index.js

```diff
@@ -59,18 +95,26 @@ function readDirectory(dirName, stream) {
   return files;
 }
 
-// If stream is true, a stream is provided instead of a buffer for
-// each file object along with the corresponding size
-function getDefaultProjects(stream) {
-  var DefaultProjects = {};
-  var projects = fs.readdirSync(__dirname);
+function cacheProjectFiles() {
+  // The folder containing this index.js file also contains
+  // the default content, each in its own folder. We get a directory
+  // listing and remove this index.js file, leaving only the default
+  // content folders to iterate through.
+  var projects = doSyncOp(fs.readdirSync.bind(null, __dirname));
```

Comment:
Are you sure this is right? I'm pretty sure you should be binding `fs` not `null`